docs/migrations/v2-to-v3.md

```diff
@@ -1,4 +1,4 @@
-# Migrating from v2.0.0 to v3.0.0
+# Migrating from v2.x.x to v3.0.0
```

Comment:
I changed this so that it's consistent with the title for the migration from v1 to v2. Maybe we should also simplify it a just say `v2 to v3` since this migration applies if you're migrating from any version of the v2 line to any version of the v3 line?

Reply:
sounds good to me!